src/features/staking/utils/helpers.ts

```diff
@@ -112,7 +112,7 @@ export const handleButtonActions = ({
   approve: (() => Promise<WriteContractResult>) | undefined;
   stake: (() => void) | undefined;
   unstake: (() => void) | undefined;
-  setValue: UseFormSetValue<{ stakingAmount: number }>;
```

Comment:
Same as mentioned above - these are small changes that were made in the "merge fixes" branch, so these won't get merged again into `main`